### ISSUE TITLE
Map Ins key to something 

### ISSUE THREAD

**Issue by emendelson** [Gcenx/WineskinServer#177]
Apologies if the answer is obvious and I'm just wasting bandwidth. Is there a way to map the PC Insert key to a Mac keystroke in Wineskin? In DOSBox, I can use Cmd-DownArrow for Ins, but I can't see any way to do this for Wineskin. Any advice will be very welcome.

**Comment by Gcenx** [comment]
If your not using a full-size keyboard the macOS key combination would commonly be “fn enter”

I’ve not dig into wine keyboard functions to see if there’s someway to change around the keyboard mapping.

**Comment by emendelson** [comment]
Interesting about fn-Enter. It doesn't seem to work with the applications I'm trying (WordPerfect for DOS in multiple versions) but that could just be WordPerfect weirdness. 

Years and years ago I figured out how to patch wineskin.drv.so, but then Wineskin included the option I was looking for, so I stopped working on it. Here's the page I wrote back then, but it's probably not of any use today:

http://www.columbia.edu/~em36/winekeymap.html
